.changeset/twenty-masks-exist.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-catalog-backend-module-aws': minor
+---
+
+AwsEksClusterProcessor supports Entity callback function
```

Comment:
We don't need two changesets here, can we consolidate them? Since it's not a bug fix, it shouldn't be a patch.

Reply:
Hello @Edje-C - patch one is a bug fix (the region isn't being used previously. The PR description has more information), whereas this one is adding an entity callback function. Thanks!

Reply:
Oh okay. I didn't realized it was required, I guess I missed that when reading the desc?

This still needs to be 1 minor changeset since we're adding functionality.

Reply:
@pjungermann curious your thoughts on this as you suggested having 2 different changesets

Reply:
The feedback to consolidate the changesets is from the backstage maintainers and this PR will not be merged with two 🤙🏾

Reply:
Updated! @Edje-C